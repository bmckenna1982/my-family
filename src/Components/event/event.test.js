import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Event from './event';

describe('Event component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Event />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders the UI as expected without day', () => {
    const tree = renderer
      .create(<Event startTime='3pm' title='Baseball practice' />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  });

  it('renders the UI as expected with day', () => {
    const tree = renderer
      .create(<Event day='Today' startTime='3pm' title='Baseball practice' />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  });
})


