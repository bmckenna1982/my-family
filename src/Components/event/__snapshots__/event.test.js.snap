// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event component renders the UI as expected with day 1`] = `
<div
  className="event-container"
>
  <div
    className="event-day"
  >
    Today
     
  </div>
  <div
    className="event-time"
  >
    3pm
  </div>
  <div
    className="event-title"
  >
    Baseball practice
  </div>
</div>
`;

exports[`Event component renders the UI as expected without day 1`] = `
<div
  className="event-container"
>
  <div
    className="event-day"
  >
     
  </div>
  <div
    className="event-time"
  >
    3pm
  </div>
  <div
    className="event-title"
  >
    Baseball practice
  </div>
</div>
`;
